Implement GeoIP feature (#14)
k8gb-io/coredns-crd-plugin: README.md

}
```

## Resolving order

> In case dnsendpoint object's target have label `strategy: geoip` CoreDNS k8s_crd plugin will respond in a special way:

Reviewer: ```suggestion
In case dnsEndpoint object's target has a label of `strategy: geoip` CoreDNS `k8s_crd` plugin will respond in a special way:
```

> * assuming record have multiple IPs associated with it, and dns Message comes with edns0 CLIENT-SUBNET option.

Reviewer: ```suggestion
* Assuming record has multiple IPs associated with it, and DNS message comes with edns0 `CLIENT-SUBNET` option.
```

> * CoreDNS will compare "DC" tag for IP exctracted from CLIENT-SUBNET option against available endpoint.targets

Reviewer: ```suggestion
* CoreDNS will compare `DC` tag for IP extracted from `CLIENT-SUBNET` option against available Endpoint.Targets
```

* Return only IPs where tags match

> * If IP have no common tag, all entries are returned.

Reviewer: ```suggestion
* If IP has no common tag, all entries are returned.
```

> * CoreDNS must be supplied with specially crafted GeoIP database in MaxMind DB format and mounted as `/geoip.mmdb` Refer to `terratest/geo` for examples.

Reviewer: ```suggestion
* CoreDNS must be supplied with a specially crafted GeoIP database in MaxMind DB format and mounted as `/geoip.mmdb` Refer to `terratest/geo` for examples.
```
